### DISCUSSION TITLE
Is it possible to change the root path of the admin UI?

### DISCUSSION THREAD

**Discussion by benjoshua** [keystonejs/keystone#7746] [Questions] [answered]
i'll be happy to host my public website in the `/` path and enter the admin ui in `/admin` or something like that. 
Is it possible?
Thanks!

**Comment by nedgar** [comment, marked as answer]
I was looking for this too. I don't see it in the doc on the main site, but it is possible. Try adding this to your keystone config:
```js
{
...
  ui: {
    isDisabled: false,
    basePath: '/admin',
  }
}
```
See the [AdminUIConfig type](https://github.com/keystonejs/keystone/blob/41276b6877c0c1e9332d8c3e37ccd772a5b2c48b/packages/core/src/types/config/index.ts#L165).

**Reply by dcousens** [reply]
Reference https://github.com/keystonejs/keystone/pull/8545
